server/README.md

```diff
@@ -23,9 +23,13 @@
     - Currently, RemoteController.cs is attached to the RemoteController GameObject
 
```

Comment:
at like line 17 we can change to server.ts if you want, ow i can do on my own next commit!